### PR TITLE
Deprecate the SEV demo documentation

### PR DESCRIPTION
**Pull request by UnmeshDeodhar**
Deprecate the SEV demo document and add a link to latest instructions.

Signed-off-by: Unmesh Deodhar <udeodhar@amd.com>

### CODE REVIEW

File: demos/sev-demo/README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# DEPRECATED
+
+Please visit this page for the most up to date instructions: https://github.com/confidential-containers/documentation/blob/main/quickstart.md#sev
```
Comment (fitzthum) [inline demos/sev-demo/README.md:3]: Can we add something about why we are leaving this document and not just deleting it, like it's an overview of the changes that are made for SEV or something like that.

**Review by fitzthum — APPROVED** [review]
Looks fine, although I am open to any other suggestions about what we should do with this.

I think we should extend the note somewhat. See comment.

**Comment by fidencio** [comment]
@UnmeshDeodhar, wouldn't be worth it removing the instructions entirely considering there's no more use for those?

File: demos/sev-demo/README.md (hunk 1 above)
Comment (fidencio) [reply demos/sev-demo/README.md:3]: I'd be in favour of actually removing the content, as it may end up confusing whoever is trying to use it.

Comment (fitzthum) [reply demos/sev-demo/README.md:3]: Yeah I am fine with deleting.

**Comment by UnmeshDeodhar** [comment]
@fidencio @fitzthum thanks for reviewing. Updated the changes to delete the file entirely to avoid confusion.

**Comment by fidencio** [comment]
After privately talking to @UnmeshDeodhar, I've force-pushed to his branch in order to drop the merge commit and keep only the commit related to dropping the outdated docs.

LGTM, thanks @UnmeshDeodhar!

**Review by fidencio — APPROVED** [review]

**Pull request merged**